apps/docs/app/features/portable-text/PortableText.tsx

```diff
@@ -156,43 +156,53 @@ const components: Partial<PortableTextReactComponents> = {
         </Flex>
       );
     },
-    imageWithCaption: ({ value }) => (
-      <Box>
-        {value.image && (
-          <Box>
-            <Image
-              src={urlBuilder
-                .image(value.image)
-                .width(924)
-                .fit("max")
-                .auto("format")
-                .url()}
-              alt={value.alt || ""}
-              mx="auto"
-              mt={2}
-            />
-          </Box>
-        )}
-        {value.caption && (
-          <Stack textStyle="xs" color="alias.dimGrey" mt={-1}>
-            <PortableText value={value.caption} />
-          </Stack>
-        )}
-      </Box>
-    ),
-    image: ({ value }) => (
-      <Image
-        src={urlBuilder
-          .image(value.image)
-          .width(924)
-          .fit("max")
-          .auto("format")
-          .url()}
-        alt={value.alt}
-        mx="auto"
-        mt={2}
-      />
-    ),
+    imageWithCaption: ({ value }) => {
+      const [, , dimensions] = value.image.asset._ref.split("-");
+      const aspectRatio = dimensions.split("x").join(" / ");
+      return (
+        <Box>
+          {value.image && (
+            <Box>
+              <Image
+                src={urlBuilder
+                  .image(value.image)
+                  .width(924)
+                  .fit("max")
+                  .auto("format")
+                  .url()}
+                alt={value.alt || ""}
+                mx="auto"
+                mt={2}
+                __css={{ aspectRatio }}
```

Comment:
I need to use the `__css` prop here, because it's not yet supported by Chakra UI.